public/404.html

```diff
@@ -3,6 +3,7 @@
 <html>
   <head>
   <meta charset=utf-8>
+  <meta name="robots" content="noindex">
```

Comment:
added these to prevent google and other search sites from including these error pages in search results